test/e2e/storage/volume_provisioning.go

```diff
@@ -54,57 +54,83 @@ import (
 const (
 	// Plugin name of the external provisioner
 	externalPluginName = "example.com/nfs"
+	// Number of PVCs for multi PVC tests
+	multiPVCcount = 3
 )
 
 func testBindingWaitForFirstConsumer(client clientset.Interface, claim *v1.PersistentVolumeClaim, class *storage.StorageClass) (*v1.PersistentVolume, *v1.Node) {
+	pvs, node := testBindingWaitForFirstConsumerMultiPVC(client, []*v1.PersistentVolumeClaim{claim}, class)
+	return pvs[0], node
+}
+
+func testBindingWaitForFirstConsumerMultiPVC(client clientset.Interface, claims []*v1.PersistentVolumeClaim, class *storage.StorageClass) ([]*v1.PersistentVolume, *v1.Node) {
 	var err error
 
 	By("creating a storage class " + class.Name)
 	class, err = client.StorageV1().StorageClasses().Create(class)
 	Expect(err).NotTo(HaveOccurred())
 	defer deleteStorageClass(client, class.Name)
 
-	By("creating a claim")
-	claim, err = client.CoreV1().PersistentVolumeClaims(claim.Namespace).Create(claim)
-	Expect(err).NotTo(HaveOccurred())
+	By("creating claims")
+	var createdClaims []*v1.PersistentVolumeClaim
+	var claimNames []string
```

Comment:
The functions used below like `WaitForPersistentVolumeClaimsPhase`, `CreatePod` need access to the slices `createdClaims` and `claimNames`. So putting them into a map will require extracting the keys and values as slices multiple times down below. To avoid doing that and given we don't need to look up claim through names, I am keeping it as two separate slices that can be directly passed to the functions used below.